Makefile

```diff
@@ -68,7 +69,7 @@ KEYCLOAK_AUTH_SERVER_URI = https://common-logon-test.hlth.gov.bc.ca/auth
 endif
 
 # Keycloak ref variables
-export KEYCLOAK_REDIRECT_URI ?= https://${DOMAIN}/
+export KEYCLOAK_REDIRECT_URI = https://${DOMAIN}/
```

Comment:
I am curious about this change. Is it desirable to override KEYCLOAK_DIRECT_URI if its already been set?

Reply:
well, you're right. This isn't exactly required. I had to constantly switch .env while deploying on dev vs running locally. But now as I check other vars, its best to keep it consistent with others and revert to ?=.
Thanks for spotting!